__testfixtures__/imports/breadcrumb.output.js

```diff
@@ -0,0 +1,11 @@
+import PubSub from 'lib/PubSub';
+var MyClass = Class(PubSub, function () {
+  this.init = function () {};
+});
+
+import BackingExtension from './layout/BackingExtension';
+exports.something = BackingExtension;
+
+var uiKeyboardTypes = {};
+import ...ui.keyboardTypes;
```

Comment:
I purposefully left this one, if there is any conflict with variables in the existing file, we will want the mods to warn / fail.  Is there a specific way to have "do not handle" test cases?